docker/pulsar-all/pom.xml

```diff
@@ -119,57 +111,73 @@
             </executions>
           </plugin>
           <plugin>
-            <groupId>com.spotify</groupId>
-            <artifactId>dockerfile-maven-plugin</artifactId>
-            <version>${dockerfile-maven.version}</version>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>${exec-maven-plugin.version}</version>
             <executions>
               <execution>
-                <id>default</id>
+                <id>docker-build</id>
+                <phase>package</phase>
                 <goals>
-                  <goal>build</goal>
-                </goals>
-              </execution>
-              <execution>
-                <id>add-no-repo-and-version</id>
-                <goals>
-                  <goal>tag</goal>
-                </goals>
-                <configuration>
-                  <repository>pulsar-all</repository>
-                  <tag>${project.version}</tag>
-                </configuration>
-              </execution>
-              <execution>
-                <id>add-no-repo-and-latest</id>
-                <goals>
-                  <goal>tag</goal>
+                  <goal>exec</goal>
                 </goals>
                 <configuration>
-                  <repository>pulsar-all</repository>
-                  <tag>latest</tag>
+                  <environmentVariables>
+                    <DOCKER_BUILDKIT>1</DOCKER_BUILDKIT>
+                  </environmentVariables>
+                  <executable>docker</executable>
+                  <workingDirectory>${project.basedir}</workingDirectory>
+                  <arguments>
+                    <argument>build</argument>
+                    <argument>--no-cache</argument>
+                    <argument>--build-arg</argument>
+                    <argument>PULSAR_TARBALL=target/pulsar-server-distribution-${project.version}-bin.tar.gz</argument>
```

Comment:
I think we need to pass `PULSAR_OFFLOADER_TARBALL` and `PULSAR_IO_DIR` instead of `PULSAR_TARBALL` for `puslar-all/build`. In fact, I think this is the root cause of `bin/sh -c mv /apache-pulsar-offloaders-*/offloaders /offloaders` error.